runtime/compiler/control/rossa.cpp

```diff
@@ -1411,6 +1411,7 @@ onLoadInternal(
 
    codeCacheConfig._allowedToGrowCache = (javaVM->jitConfig->runtimeFlags & J9JIT_GROW_CACHES) != 0;
    codeCacheConfig._lowCodeCacheThreshold = TR::Options::_lowCodeCacheThreshold;
+   codeCacheConfig._highCodeCacheOccupancyThresholdInBytes = TR::Options::_highCodeCacheOccupancyPercentage * jitConfig->codeCacheTotalKB * 10;
```

Comment:
Where does `*10` come from? Shouldn't that be `/100`?

Reply:
Also, I just realized that `_highCodeCacheOccupancyThresholdInBytes` is in bytes while `jitConfig->codeCacheTotalKB` is in KB. We may need to shift right by 10 to transform those KB into bytes.

Reply:
I had forgotten that the conversion was supposed to be a right shift. The `*10` was `*1000` then `/100`.